src/utils/db.js

```diff
@@ -0,0 +1,10 @@
+import { Sequelize } from 'sequelize';
+import 'dotenv/config';
+
+export const client = new Sequelize({
```

Comment:
The Sequelize constructor is missing the 'database' parameter as the first argument. It should be in the format: new Sequelize(database, username, password, { ...options }).